### PR TITLE
[Filebrowser] Fix support for reading Parquet files

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- Fix support for reading the parquet files in filebrowser
- Using Pyarrow for reading the parquet files

## How was this patch tested?

- Locally
- Unittest

### CODE REVIEW

File: apps/filebrowser/src/filebrowser/views.py [hunk 1]
```diff
@@ -984,12 +985,15 @@ def _read_avro(fhandle, path, offset, length, stats):
 def _read_parquet(fhandle, path, offset, length, stats):
   try:
     size = 1 * 128 * 1024 * 1024  # Buffer file stream to 128 MB chunks
-    data = string_io(fhandle.read(size))
 
-    dumped_data = string_io()
-    parquet._dump(data, ParquetOptions(limit=1000), out=dumped_data)
```
Comment (Harshg999) [inline apps/filebrowser/src/filebrowser/views.py:990]: Remove `ParquetOptions` related dead code also?

File: apps/filebrowser/src/filebrowser/views.py [hunk 2]
```diff
@@ -1061,9 +1065,14 @@ def detect_snappy(contents):
 def detect_parquet(fhandle):
   """
   Detect parquet from magic header bytes.
-  Python 2 only currently.
   """
-  return False if sys.version_info[0] > 2 else parquet._check_header_magic_bytes(fhandle)
+  # Parquet files start with a specific 4-byte magic number: 'PAR1'
+  PARQUET_MAGIC_NUMBER = b'PAR1'
```
Comment (Harshg999) [inline apps/filebrowser/src/filebrowser/views.py:1070]: Nit: Constant at top?

File: apps/filebrowser/src/filebrowser/views.py [hunk 3]
```diff
@@ -984,12 +985,15 @@ def _read_avro(fhandle, path, offset, length, stats):
 def _read_parquet(fhandle, path, offset, length, stats):
   try:
     size = 1 * 128 * 1024 * 1024  # Buffer file stream to 128 MB chunks
```
Comment (Harshg999) [inline apps/filebrowser/src/filebrowser/views.py:987]: Nit: Change comment to 128 MiB

**Review by Harshg999 — APPROVED** [review]
LGTM! Just few nits..

**Review by JohanAhlen — APPROVED** [review]
LGTM and +1 on Harsh's comments.

**Pull request merged**